.license_scout.yml

```diff
@@ -257,6 +257,8 @@ exceptions:
       reason: (sd) Binary use only. Already shipped in chef-server. Exception made by Chef Legal
     - name: chef/automate-openjdk
       reason: Exception made by Chef Legal
+    - name: core/nss-myhostname
+      reason: (dan) linked only in ruby interpreter.
```

Comment:
core/nss-myhostname isn't being linked in ruby, it's a library for nss/glibc.

Reply:
Correct, I learned this between when I wrote this comment and when I wrote the PR description. Will update.